Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/FormElements/UtilityAssociationsFormElement/AssociationRemovalConfirmation.swift

```diff
@@ -52,32 +50,19 @@ private struct AssociationRemovalConfirmationDialog: ViewModifier {
     let onRemoval: () -> Void
     
     func body(content: Content) -> some View {
-        if isPortraitOrientation {
-            content
-                .confirmationDialog(
-                    confirmationTitle,
-                    isPresented: $isPresented,
-                    titleVisibility: .visible
-                ) {
-                    actions
-                } message: {
-                    confirmationMessage
-                }
```

Comment:
In iOS 26, confirmation dialogs adopt the popover style and have a misplaced arrow when used on list items.

![Simulator Screen Recording - iPhone 17 - 2025-09-17 at 15 05 33](https://github.com/user-attachments/assets/3bb691f0-0e37-45c8-82c2-55ff5aebe188)

 In iOS 18 this is only an issue on iPadOS. 

Choosing to go with alerts instead for both platforms now as a result.